Source/Controllers/BundleSupport/SPBundleEditorController.m

```diff
@@ -964,7 +964,7 @@ - (BOOL)saveBundle:(NSDictionary*)bundle atPath:(NSString*)aPath
 																   error:&error] retain];
 			
 			if(!cmdData || error) {
-				NSLog(@"“%@” file couldn't be read. (error=%@)", cmdFilePath, error);
+				SPLog(@"“%@” file couldn't be read. (error=%@)", cmdFilePath, error);
 				NSBeep();
```

Comment:
What if we got rid of the beeps entirely from the bundle stuff? Instead of tracking if things beeped and beeping somethings. Unless we're showing error messages, the beeps don't seem to serve any real purpose?

Reply:
I'm OK to switch off, but I have sounds switched off already. Others might like the beep to notify of error?

Reply:
could make it a preference, but then, that's more prefs...

Reply:
I think this PR's target should be to fix the beeping when ser doesn't know why it beeps, but not necessaril reduce the beeps where they make sense... :)

Reply:
I agree with @Kaspik. I think it only makes sense to beep when it's clear what errored. If it errored behind the scenes and didn't notify the user of a bundle thing I would just not beep!

Reply:
There's 109 NSBeeps(). I've handled the bundle beeps. That will do for this PR I think...